outputs.tf

```diff
@@ -0,0 +1,34 @@
+output "map_configs" {
+  value       = module.yaml_config.map_configs
+  description = "Terraform maps from YAML configurations"
+}
+
+output "list_configs" {
+  value       = module.yaml_config.list_configs
+  description = "Terraform lists from YAML configurations"
+}
+
+output "all_imports_list" {
+  value       = module.yaml_config.all_imports_list
+  description = "List of all imported YAML configurations"
+}
+
+output "all_imports_map" {
+  value       = module.yaml_config.all_imports_map
+  description = "Map of all imported YAML configurations"
+}
```

Comment:
What's the case for having the imports output as a map?

Reply:
I don't know the exact use-case, but it shows some kind of a hierarchy - not sure if we need that